.github/workflows/tools/dispatch-build.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
```

Comment:
let's strip `.py` suffix -- there is a shebang, and I do not see us invoking from windows, and windows people could just prepend with `python`. This is a typical recommendation in Debian land and, unless it is also to be used as python module, I tend to follow it

Reply:
and shouldn't this script be used in `update-mirror.yml` (again - to not duplicate code; might need to get python  in the env if not there yet)

Reply:
Done and done.